components/camel-sql/src/main/java/org/apache/camel/component/sql/stored/template/generated/SSPTParserConstants.java

```diff
@@ -1,19 +1,3 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
 /* Generated By:JavaCC: Do not edit this line. SSPTParserConstants.java */
 package org.apache.camel.component.sql.stored.template.generated;
```

Comment:
Here we lost the ASF license header, perhaps revert this change?

Reply:
Hello Mr.Zoran I have no idea what caused to this, I just ran ./mvnw -Pfastinstall install. I'll do a revert

Reply:
Hello, Mr.Zoran I reverted that whole commit, can you check now?